modules/openapi-generator/src/main/resources/python-flask/model.mustache

```diff
@@ -62,15 +62,17 @@ class {{classname}}(Model):
 {{/vars}}
 
     @classmethod
-    def from_dict(cls, dikt){{^supportPython2}} -> '{{classname}}'{{/supportPython2}}:
```

Comment:
Why did you remove this code which works for python 2.7?
We still need this because we support python 2.7 and higher.
Can you add it back in?